ecosystem/sep-0001.md

```diff
@@ -116,9 +116,10 @@ These fields go in the `stellar.toml` `[[CURRENCIES]]` list, one set of fields f
 
 Field | Requirements | Description
 ------|--------------|------------
-code | string (<= 12 char) | Token code
+code | string (<= 12 char) | Asset code. Required for assets that are issued using a Stellar account as opposed to a contract.
+issuer | `G...` string | Asset issuer Stellar public key. Required for assets that are issued using a Stellar account as opposed to a contract.
+contract | `C...` string | Contract ID of the asset contract. The asset must be compatible with the [SEP-41 Token Interface](sep-0041.md) to be defined here.
```

Comment:
Should this be limited to assets, which are classic Stellar, or should it support custom tokens?

We're using language here like assets but I think we're using it to refer to also include custom tokens. 

As much as possible as should only use the term asset for stellar assets, and use the term token for the superset that includes stellar assets and custom tokens implementing the interface.

Reply:
> As much as possible we should only use the term asset for stellar assets, and use the term token for the superset that includes stellar assets and custom tokens implementing the interface

Is that definition consensus among the ecosystem? I always thought of tokens to be "not Stellar assets" meaning non-SAC tokens, whether they're compatible with the SEP-41 token interface or not.

> should it support custom tokens?

I don't think it is useful to list custom tokens that don't adhere to a standardized interface, since the client won't know how to interface with the contract. Thats why I said it must be compatible with the SEP-41 token interface. 

Maybe we should add another field like `contract_interface` with its supported values being standard token interfaces like SEP-41 or a future NFT token interface, for example.

Reply:
When communicating with the Stellar Asset contract developers use the token client, so I think tokens definitely include assets. We have pretty uniformly used the term "custom token" with the word "custom" when referring to implementations of tokens that are exclusively not an asset.

For the most part contract devs should dev for tokens, that will support both.

Reply:
cc @tomerweller @namankumar regarding confirming this nomenclature isn't just me.

Reply:
>custom tokens that don't adhere to a standardized interface, since the client won't know how to interface with the contract. Thats why I said it must be compatible with the SEP-41 token interface.

To be clear, when we say custom tokens, we're referring to any contract that implements the SEP-41 token interface that isn't the asset contract.

---

token = any contract that implements SEP-41
custom token = any contract that implements SEP-41 excluding the stellar asset contract

Reply:
What is our term for non-SEP41 tokens like NFTs?

Reply:
I don't think a term has arisen for them yet.